samples/highcharts/demo/area-missing/demo.js

```diff
@@ -1,32 +1,33 @@
+// Data retrieved from https://www.ssb.no/statbank/table/10467/
 Highcharts.chart('container', {
     chart: {
         type: 'area'
     },
     title: {
-        text: 'Fruit consumption *'
+        text: 'Born persons, by boys\' name'
     },
     subtitle: {
-        text: '* Jane\'s banana consumption is unknown',
+        text: '* The name Yasin was not given this year.',
```

Comment:
This explanation indicates there were 0 instances of Yasin this year. But there is a fundamental difference between 0 and null, where 0 is a real value that would be plotted in the area chart, and `null` is missing data. Better change the text to `* Missing data for Yasin this year`